### ISSUE TITLE
[Win 11] Preview is broken

### ISSUE THREAD

**Issue by Shouri14** [Benexl/FastAnime#25]
Anime poster image section is broken,
![image](https://github.com/user-attachments/assets/e3dde23b-c1ce-45e5-93bc-86e4594653a7)

Info:
Version : `v2.7.9`
Terminal Emulator : `Windows Terminal Preview` App

Reproduce : `fastanime --icons --preview --fzf anilist`

**Comment by Shouri14** [comment]
Seems to be from,
https://github.com/Benex254/FastAnime/blob/4f0768a0609b7ab8aefbc81f416b890d23fcb092/fastanime/cli/utils/scripts.py#L13

Looks like some problem with how powershell handles backlash.
I tried breaking down the command, but didn't work.
I'll try again when I can.

**Comment by Rashify** [comment]
![image](https://github.com/user-attachments/assets/e8a91ceb-af77-44a5-9289-22cfb49e4e21)
I get this

**Comment by ykhan21** [comment]
@Shouri14, this is not a powershell script so it shouldn't work in powershell. I believe it needs a bash shell to run this script. You can try editing this line
https://github.com/Benex254/FastAnime/blob/4f0768a0609b7ab8aefbc81f416b890d23fcb092/fastanime/cli/interfaces/utils.py#L260
so that on Windows it resolves bash to `C:/Program Files/Git/bin/bash.exe` or `C:/Users/USERNAME/scoop/apps/git/current/bin/bash.exe`, assuming that you have git.

@Rashify, your error might indicate that the bash that it is picking up is WSL. I don't think WSL bash can run that file like that, so you can try running fastanime in WSL instead.

**Comment by Benex254** [comment]
Join the discord, for help it is not broken

**Comment by Rashify** [comment]
@ykhan21 Thank you so much for your help I figured out I didn't even need to edit the code, all I had to do was add "C:\Program Files\Git\bin" to "Path" in my environment variables!

**Comment by ykhan21** [comment]
An update might make it so that WSL bash is on the path with higher priority than Git for Windows bash. This is why it works for some people sometimes and is why you need to edit the code to make sure it picks up Git for Windows bash, even if you don't have WSL installed.

ani-cli users have had this problem for a while.
________________________________

**Comment by Benexl** [comment]
Okey, i see you guys really suffering on fake linux lol. I'll be waiting for you guys pr then. Never realized that somehow `shutil.which` won't be accurate in wsl. But i prefer a solution thats not hard coded path, just incase.

**Comment by Benexl** [comment]
actually i just noticed @Rashify gave a solution, that failed for you?
Cause i think this is still more a wsl issue than an issue with the code

**Comment by ykhan21** [comment]
@Benexl, fastanime might be fine on WSL, but this doesn't concern WSL.

The problem occurs when you use fastanime on Windows. fastanime needs a bash. When you "which" bash on Windows (NOT WSL), you'll get the path to the WSL bash executable, because MS put a bash.exe on the path which is fact WSL. This won't work for your script because of the two different file systems (WSL and Windows). To get your script to work on Windows, simply use the bash executable that comes with Git for Windows (GfW), which is a native Windows shell with a posix environment.

The thing is, fastanime on Windows will work out of the box for some people because they might have GfW bash on their path, so "which" bash will return the correct bash. However, they will experience a breakage at some point when an update occurs where MS will put WSL bash on a higher priority than the GfW bash.

TLDR: Change the python code so that if you're on Windows, you pick the GfW bash.
________________________________

**Comment by Benexl** [comment]
Yeah am gonna close this again. Am not even in wsl or windows how do you expect me to do those changes without testing. And for those who are there none have complained of anything of the sort. Please join the discord and as for help cause am starting to think again there is no issue.
